### DISCUSSION TITLE
GitHub is now forcing me to use my full real name on my profile?

### DISCUSSION THREAD

**Discussion by ThePat02** [community/community#140645] [GitHub Education]
### Hi everyone,

According to this error message on my reapplication, GitHub needs me to put my real name on my public profile to continue using the benefits?

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

There should be no reason to deny me the application just because I don't want to give my full name on my PUBLIC PROFILE. In addition, digital copies of my enrollment paper are not accepted as well? Do I need to print it out and snap a picture?

**Comment by Anit-K-Peter** [comment]
enrollment paper are not accepted, then yes, you would likely need to print out the document and take a clear picture of it.

**Reply by ThePat02** [reply]
Sorry, I didn't choose the correct word. In German it is called "Studiumsblatt", which basically is a document that verifies that O actually study at the university at the time the document was requested. It is used for any official verification and I used it on GitHub before.

**Comment by Bluemethyst** [comment]
I just got this as well! I submitted with a digital PDF and not using my real name and it said i got rejected, im going to try again with a photo of my admission letter

**Comment by sxck36** [comment]
I'm facing the same issue, there's not even an option to verify through a support ticket anymore. Why would I want my full name displayed on a public profile?

**Reply by Bluemethyst** [reply]
I got it okay, I just printed out my proof on enrollment and took a photo instead

**Reply by sxck36** [reply]
Yeah, I actually just got approved. I just added my school email to my account and verified it. Didn't need to add my name to my profile.

**Reply by ThePat02** [reply]
I had to temporarily change my account name. A friend of mine even had to do other things too. The automatic system is flawed.

**Comment by HumanoidSandvichDispenser** [comment]
I put my real name on my billing info instead and it worked.
